pages/first-project.mdx

```diff
@@ -0,0 +1,362 @@
+import { Callout, Cards, Steps, Tabs } from "nextra/components";
+
+# **Getting Started: Create Your First Project**
```

Comment:
We do not to make the headings bold. so remove the ** around the titles and headings.